docs/sql-ref-syntax-ddl-create-table-datasource.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: global
+title: SHOW CREATE TABLE
+displayTitle: SHOW CREATE TABLE
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `CREATE TABLE` statement creates a new table using a Data Source. 
+
+### Syntax
+{% highlight sql %}
+CREATE TABLE [IF NOT EXISTS] [db_name.]table_name
+  [(col_name1 col_type1 [COMMENT col_comment1], ...)]
+  USING datasource
+  [OPTIONS (key1=val1, key2=val2, ...)]
+  [PARTITIONED BY (col_name1, col_name2, ...)]
+  [CLUSTERED BY (col_name3, col_name4, ...) INTO num_buckets BUCKETS]
+  [LOCATION path]
+  [COMMENT table_comment]
+  [TBLPROPERTIES (key1=val1, key2=val2, ...)]
+  [AS select_statement]
+{% endhighlight %}
+
+### Parameters
+
+<dl>
+  <dt><code><em>USING datasource</em></code></dt>
+  <dd>Datasource using which the table is created.Data source can be CSV, TXT, ORC, JDBC,PARQUET, etc.</dd>
```

Comment:
This needs some proofreading, here and below. Space needs to follow punctuation. 
"Data source" needs to be consistent and refer to the argument above.
"using which the table is created" -> used to create the table.
But can you say any more about what this means? This isn't really adding much documentation.